packages/common/src/Adaptor.js

```diff
@@ -267,12 +267,32 @@ export function asData(data, state) {
  * the state's references.
  * @public
  * @function
- * @example
- * each("$.[*]",
- *   create("SObject",
- *     field("FirstName", sourceValue("$.firstName"))
+ * @example <caption>Inserting patitent data using lazy state. (Only in v2)</caption>
+ * each($.data,
+ *   insert("patient",
```

Comment:
As you pointed out on slack, the data thing is weird.

What about we add a comment?
```
each($.data,
   // Inside the callback operation, `$.data` is scoped to the item under iteration
   insert("patient",
     {
       patient_name: $.data.properties.case_name,
       patient_id: $.data.case_id
     }
  )
 )

Reply:
I have update the example with a comment